### PR TITLE
Vfy default response

### PR DESCRIPTION
**Pull request by sshajiya**
In this PR NAP Default Page Verification is added

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: Cloud Formation Template
+
+on:
+  push: {branches: master}
+  pull_request: {branches: main}
```
Comment (yoctoalex) [inline .github/workflows/pipeline.yml:5]: There is no main branch. Should be master.

File: delete_stack.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```
Comment (yoctoalex) [inline delete_stack.py:1]: move to scripts/pipeline

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```
Comment (yoctoalex) [inline utils.py:1]: move to scripts/pipeline

File: test1.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```
Comment (yoctoalex) [inline test1.py:1]: move to scripts/pipeline
rename to basic_page_verification_test.py or similar

**Review by yoctoalex — CHANGES_REQUESTED** [review]

File: .github/workflows/pipeline.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        run: |
+          python -m pip install --user boto3
+          python -m pip install --user requests
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
+          aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+          aws-region: ${{secrets.AWS_REGION}}
```
Comment (464d41) [inline .github/workflows/pipeline.yml:34]: Likely AWS creds are avaliable as env vars via github secrets, therefore, this part is redundant. Please check and remove if so.

File: .github/workflows/pipeline.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      - name: Launch the Stack
+        uses: ShahradR/action-taskcat@v1
+        with:
+          commands: test run -n
+          update_cfn_lint: true
```
Comment (464d41) [inline .github/workflows/pipeline.yml:39]: Template format check should be complete by this time. No need to check here.

File: .github/workflows/pipeline.yml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+          python -m pip install --user requests
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
+          aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+          aws-region: ${{secrets.AWS_REGION}}
+      - name: Launch the Stack
+        uses: ShahradR/action-taskcat@v1
```
Comment (464d41) [inline .github/workflows/pipeline.yml:36]: What is the reason to use taskcat here? Why not just a raw awscli?

**Review by 464d41 — CHANGES_REQUESTED** [review]

File: test1.py (hunk 4 above)
Comment (sshajiya) [reply test1.py:1]: Sure

File: .github/workflows/pipeline.yml (hunk 7 above)
Comment (sshajiya) [reply .github/workflows/pipeline.yml]: removed taskcat and added the aws cli code.

**Pull request closed without merging**